python/semantic_kernel/utils/settings.py

```diff
@@ -246,3 +246,33 @@ def azure_aisearch_settings_from_dot_env() -> Tuple[str, str]:
     assert api_key is not None, "Azure AI Search API key not found in .env file"
 
     return api_key, url
+
+
+def azure_aisearch_datasource_settings_from_dot_env() -> Tuple[str, str, str]:
```

Comment:
there already is a azure_aisearch_settings_from_dot_env method, why not extend that with a parameter to also get the index?

Reply:
good point. added to that one, also added a method to directly deserialize to the AI search data source class that can be used in the azure chat with data settings.